UNITTESTS/stubs/EventFlags.cpp

```diff
@@ -0,0 +1,24 @@
+#include "rtos/EventFlags.h"
```

Comment:
`ATHandler_stub` - has suffix, but some of the stubs do not have, like this one. Is there a naming convention?